gulpfile.js/tasks/stylesheets.js

```diff
@@ -1,14 +1,16 @@
 if(!TASK_CONFIG.stylesheets) return
 
-var gulp         = require('gulp')
-var gulpif       = require('gulp-if')
-var browserSync  = require('browser-sync')
-var sass         = require('gulp-sass')
-var sourcemaps   = require('gulp-sourcemaps')
-var handleErrors = require('../lib/handleErrors')
-var autoprefixer = require('gulp-autoprefixer')
-var projectPath  = require('../lib/projectPath')
-var cssnano      = require('gulp-cssnano')
+var gulp             = require('gulp')
+var addPostCssPlugin = require('../lib/addPostCssPlugin')
+var autoprefixer     = require('autoprefixer')
+var browserSync      = require('browser-sync')
+var cssnano          = require('cssnano')
+var gulpif           = require('gulp-if')
+var handleErrors     = require('../lib/handleErrors')
+var postcss          = require('gulp-postcss')
+var projectPath      = require('../lib/projectPath')
+var sass             = require('gulp-sass')
+var sourcemaps       = require('gulp-sourcemaps')
```

Comment:
a little hard to read this diff because the list is now alphabetized, with gulp pinned to the top. change is adding addPostCssPlugin, autoprefixer, cssnano, and postcss